boards/arm/nrf54h20pdk_nrf54h20/doc/index.rst

```diff
@@ -0,0 +1,148 @@
+.. _nrf54h20pdk_nrf54h20:
+
+nRF54H20 PDK
+############
+
+Overview
+********
+
+The nRF54H20 PDK is a single-board preview development kit for evaluation
+and development on the Nordic nRF54H20 System-on-Chip (SoC).
+
+The nRF54H20 is a multicore SoC with:
+
+* an Arm Cortex-M33 core with DSP instructions, FPU, and Armv8-M Security
+  Extensions, running at up to 320 MHz, referred to as the **application core**
+* an Arm Cortex-M33 core with DSP instructions, FPU, and Armv8-M Security
+  Extensions, running at up to 256 MHz, referred to as the **radio core**.
+
+The ``nrf54h20pdk_nrf54h20_cpuapp`` build target provides support for
+the application core on the nRF54H20 SoC.
+The ``nrf54h20pdk_nrf54h20_cpurad`` build target provides support for
+the radio core on the nRF54H20 SoC.
+
+nRF54H20 SoC provides support for the following devices:
+
+* :abbr:`ADC (Analog to Digital Converter)`
+* CLOCK
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`GPIOTE (General Purpose Input Output tasks and events)`
+* :abbr:`GRTC (Global real-time counter)`
+* :abbr:`I2C (Inter-Integrated Circuit)`
+* MRAM
+* :abbr:`PWM (Pulse Width Modulation)`
+* RADIO (Bluetooth Low Energy and 802.15.4)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UART (Universal asynchronous receiver-transmitter)`
+* :abbr:`USB (Universal Serial Bus)`
+* :abbr:`WDT (Watchdog Timer)`
```

Comment:
I guess this list should contain only : GPIO, GPIOTE, GRTC, UART (as in the table starting from line 60)

Reply:
This is what is available in the SoC, in hardware. That tables below list what is currently supported in Zephyr.